cognite/client/data_classes/assets.py

```diff
@@ -268,6 +275,10 @@ def parent_external_id(self):
     def labels(self):
         return AssetUpdate._LabelAssetUpdate(self, "labels")
 
+    @property
+    def geoLocation(self):
```

Comment:
we prefer snake_case over camelCase
```suggestion
    def geo_location(self):
```